src/python/pants/util/contextutil.py

```diff
@@ -14,10 +14,11 @@
 import time
 import uuid
 import zipfile
+from builtins import object
```

Comment:
Temporarily removing the override of `str()` because it breaks a bunch of tests. I'll figure out how to fix this in a separate dedicated PR

Reply:
Actually for now I'm going keep the original semantics. I don't understand the ramifications to changing this. 

We might run into issues when running Python3 tests the first time - if so, we'll know we have to change the semantics from what we have now.

Below change (line 159) makes it so the semantics is the same in Py2 and Py3.